api/Permissions.json

```
@@ -1140,6 +1140,56 @@
             "deprecated": true
           }
         }
+      },
+      "speaker-selection_permission": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Permissions/speaker-selection",
+          "spec_url": "https://w3c.github.io/permissions/#dom-permissionname-speaker-selection",
+          "description": "<code>speaker-selection</code> permission",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "92"
+            },
+            "firefox_android": {
+              "version_added": "92"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
```

Experimental is usually `true` for single implementations (see [Choosing an experimental status](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status))

```suggestion
            "experimental": true,
```

---

Fair enough!